Client/YooChooseNotifier.php

```diff
@@ -200,7 +222,7 @@ protected function getContentTypeId($contentId)
 
         try {
             $contentTypeId = $this->contentService->loadContentInfo($contentId)->contentTypeId;
-        } catch (\Exception $e) {
+        } catch (Exception $e) {
```

Comment:
This was there from before, I'm not sure why it is there, but it has nothing to do with missing content type id, and all about missing content *(`loadContentInfo` is the only thing here calbe of throwing)*. Say for instance content has been deleted in between, which is particular relevant when this at later point is done asynchronously.